nftables-formula/pillar.example

```diff
@@ -0,0 +1,56 @@
+nftables:
+  config:
+    testchain:
+      # passive configurations will not be included in the top nftables.conf and can be referenced in lower "includes"
+      mode: passive
+      chains:
+        forward:
+          type: filter
+          rules:
+            - ct state established, related accept
+    sets:
+      mode: passive
+      sets:
```

Comment:
there needs to be a layer for inet/ip/ip6 here.

Reply:
Can you elaborate, do you mean some abstraction which will generate sets for the different families? Currently I just build `set`s the same way as natively in nftables (see the macros file https://github.com/openSUSE/salt-formulas/pull/61/files/730c44859d4d76bb02a4c3b17dc371161695382f), which intends specifying the family as part of the set `type`.

Reply:
how do you decide if a set should go into 

```
table ip6 nat {
}
```

```
table ip nat {
}
```

which is might be needed if you want to use sets in nat rules . those can not live in `table inet ...`

Reply:
That depends on where you include the file you define the set in. In the example pillar, a configuration file `sets.conf` is placed in `/etc/nftables.d/passive/`, which is then included in the `inet` table via line 34.